### DISCUSSION TITLE
Security Onion Install Stuck on Splash Screen

### DISCUSSION THREAD

**Discussion by ghost** [Security-Onion-Solutions/securityonion#2484] [Unsupported Versions] [answered]
Hi all, 

This is my first post and I'm sorry it say I'm posting for help. I'll start by letting you know that I am new to SO (about 2 months) and my Linux skills are beginner level.

My issue is that I have built two new bare bones computers to use as forwarding nodes and Security Onion (2.3.10 & 2.3.21) will not install on either computer. There are more nodes to build, so this is a problem. All computers are using Asus H470-I gaming motherboards with i5 6-core processors, 500GB SSDs and 16GB DDR4 RAM. I am booting the SO iso from a usb. I get to the install menu, make my selection, get the splash screen and then nothing. It just sits there on the splash screen. I've tried different menu options but it's always the same result. I disabled the SSD and tried installing on an HDD but got the same result. On both computers, I was able to successfully install and use Centos 7 and 8 without issue. I realize I can install the SO components from there but would rather have just the SO O/S installed. I also have a different, older box that I use and the same 2.3.10 iso worked fine on it.

Is there a setting in the BIOS that could be causing it? I've reset the defaults and also tried some tweaks but nothing worked.

Your thoughts/help are greatly appreciated.

Thank you,

Troy

**Comment by dougburks** [comment]
Have you tried the recommendations at https://docs.securityonion.net/en/2.3/trouble-booting.html?

**Comment by ghost** [comment]
Thanks for your reply. Yes, I’ve tried everything except for the display adapter changes. Maybe there’s a virtualization setting somewhere that I’m missing. I’ll keep looking. Thanks for your help.

**Comment by dougburks** [comment]
If you're installing directly on the hardware, then the virtualization options shouldn't matter.

You can try disabling Secure Boot. If that doesn't work, then try disabling EFI (although the current ISO should work with EFI).

Are you using Balena Etcher to create the USB?

Have you tried a different USB?

**Comment by ghost** [comment, marked as answer]
The solution was to install Ubuntu 18.04, clone from github and run the setup. Thanks for your help!
